File: problems/defc7c02-4255-47be-9812-5a9c2468fadd/go/driver.go
package main

import (
	"encoding/json"
	"log"
	"os"
)

// Update:
// Input any accordingly
// Expected any accordingly
type Testcase struct {
	Id       int  `json:"id"`
	Input    any  `json:"input"`
	Expected any  `json:"expected"`
	IsPublic bool `json:"isPublic"`
}

func loadTestcases(filename string) []*Testcase {
	file, err := os.Open(filename)
	if err != nil {
		log.Fatal("failed to open testcases file.")
	}

	var testcases []*Testcase

	decoder := json.NewDecoder(file)
	err = decoder.Decode(&testcases)
	if err != nil {
		log.Fatal("failed to decode testcases file.")
	}

	return testcases
}

File: problems/5e397e4f-e262-498c-a955-c30aa0900d52/javascript/driver.js
const fs = require('fs');

function loadTestcases(filename) {
    try {
        const data = fs.readFileSync(filename, 'utf8');
        return JSON.parse(data);
    } catch (error) {
        console.error('Failed to open testcases file.');
        process.exit(1);
    }
}

module.exports = { loadTestcases };
